### ISSUE TITLE
Installation for MacOS: use curl instead of wget

### ISSUE THREAD

**Issue by dariober** [bioconda/bioconda-utils#643]
In the installation instruction of (bio)conda for MacOS (https://bioconda.github.io/user/install.html) I suggest to replace:

```
wget https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
```

with:

```
curl -O https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
```

Since MacOS by default doesn't ship with `wget`. (It's a bit offputting for a new user to face a problem of installation for a system that is supposed to make installations easy!)

**Comment by epruesse** [comment]
Just `conda install wget`! ;-)

Seriously, good point. Can you click on the "edit on github" link on the page and make a PR? I'll merge ASAP.

**Comment by epruesse** [comment]
Fixed by #644 - thanks @dariober
